virt/virt-4-8-release-notes.adoc

```diff
@@ -49,6 +49,9 @@ The SVVP Certification applies to:
 
 //CNV-12271 A new tech preview feature allows VM owners to hot-plug and hot-unplug disks with a running virtual machine
 
+//CNV-12348 Add `--proxy-only` option to `virtctl vnc` for only proxying a VNC connection to allow manual connections
```

Comment:
The only thing I'd suggest is to avoid anthropomorphic phrasing like "The `<non-human thing>` allows you..." though I realize we do this a lot in other docs. I might rephrase this as "You can use the `--proxy-only` option with the `virtctl vnc` command to..."

Reply:
Thanks for the suggestion. I am going to merge this as is.